pkg/controllers/netpol/network_policy_controller.go

```diff
@@ -499,14 +499,31 @@ func (npc *NetworkPolicyController) ensureTopLevelChains() {
 func (npc *NetworkPolicyController) ensureExplicitAccept() {
 	// for the traffic to/from the local pod's let network policy controller be
 	// authoritative entity to ACCEPT the traffic if it complies to network policies
-	for _, filterTableRules := range npc.filterTableRules {
+	for ipFamily, filterTableRules := range npc.filterTableRules {
+		iptablesCmdHandler := npc.iptablesCmdHandlers[ipFamily]
 		for mainChain, _ := range defaultChains {
 			comment := "\"KUBE-ROUTER rule to explicitly ACCEPT traffic that comply to network policies\""
 			args := []string{"-m", "comment", "--comment", comment, "-m", "mark", "--mark", "0x20000/0x20000",
 				"-j", "ACCEPT"}
-			utils.AppendUnique(filterTableRules, mainChain, args)
-		}
-	}
+			exists, err := utils.Exists(filterTableRules, mainChain, args)
+			if err != nil {
+				utils.AppendUnique(filterTableRules, mainChain, args)
+			} else if !exists {
```

Comment:
When would we get `err != nil && !exists` to get into this case? `utils.Exists` only ever returns a nil error.